site/source/docs/compiling/Building-Projects.rst

```diff
@@ -261,12 +258,7 @@ Projects that use *configure*, *cmake*, or some other portable configuration met
 Archive (.a) files
 ------------------
 
-Emscripten supports **.a** archive files, which are bundles of object files. This is an old format for libraries, and it has special semantics - for example, the order of linking matters with **.a** files, but not with plain object files (in **.bc**, **.o** or **.so**). For the most part those special semantics should work in Emscripten, however, we support **.a** files using llvm's tools, which have a few limitations.
-
-The main limitation is that if you have multiple files in a single **.a** archive that have the same basename (for example, ``dir1/a.o, dir2/a.o``), then llvm-ar cannot access both of those files. Emscripten will attempt to work around this by adding a hash to the basename, but collisions are still possible in principle.
-
-Where possible it is better to generate shared library files (**.so**) rather than archives (**.a**) — this is generally a simple change in your project's build system. Shared libraries are simpler, and are more predictable with respect to linking.
-
+Emscripten supports **.a** archive files, which are bundles of object files. This is a simple format for libraries, that has special semantics - for example, the order of linking matters with **.a** files, but not with plain object files. For the most part those special semantics should work the same in Emscripten as elsewhere.
 
```

Comment:
maybe some notes about how to generate them? this came up in issues that I remember. like

```suggestion
To create **.a** archives, use ``emar``.
```

Reply:
Its added and example to the list below.. is that not enough do you think?